source/class/qx/tool/compiler/app/Cldr.js

```diff
@@ -88,6 +90,10 @@ qx.Class.define("qx.tool.compiler.app.Cldr", {
             encoding: "utf-8"
           })
         )
+        .catch(function (err) {
```

Comment:
Only for consistency with the above code, this could be an arrow function rather than a traditional function. I will approve anyway, but for your consideration...